store/collections/fashion.js

```diff
@@ -20,6 +21,18 @@ export const mutations = {
   enable(state) {
     state.enabled = true;
   },
+  filter(state, payload) {
```

Comment:
Unless it's envisaged this will ever operate on other fields, which I can't anticipate at present, then it's not necessary or useful to send through an array as payload. Just sends the facets, then set `state.facets` directly below.

Reply:
Also, let's rename it to `filterFacets`.